test/functionalTest/cases/0987_put_attributes/put_entity_error.test

```diff
@@ -55,8 +55,9 @@ echo
 
 echo "03. Create E1"
 echo "============="
+# FIXME P10: This creation is not legal! Maybe related with issue #1682
```

Comment:
Yeah, I believe I've seen in the code as well that /v2/entities/{eid} is able to create entities.
Probably a remnant from the old way of working.